### PR TITLE
Fix various matter-related issues

### PR DESCRIPTION
**Pull request by out-of-phaze**
## Description of changes
- Fixes single cable coils spawning with the wrong amount.
- Fixes network and electrical cables costing too much when fabricating.
- Fixes lit matches/lighters dealing brute damage instead of burn
- Fixes matches dealing 5 damage
- Fixes chewing tobacco and chewing gum being made of plastic and fiberglass
- Fix visible messages with bad deaf/self messages
- Removes rounding from matter amount modifier
- Update lighter/match object flags to fix overwriting/matter values
- Makes force-based item value calculation only apply to items with the new IS_WEAPON item flag.

## Why and what will this PR improve
Fixes the value and matter of a lot of objects being way too high. Stuff is going to be cheaper and have less matter in it across the board.

### CODE REVIEW

File: code/game/objects/items/weapons/candle/candle.dm [hunk 1]
```diff
@@ -71,7 +73,7 @@
 
 /obj/item/flame/candle/attack_self(mob/user)
 	if(lit)
-		lit = 0
+		lit = FALSE
```
Comment (comma) [inline code/game/objects/items/weapons/candle/candle.dm]: shouldn't this also change damtype back to brute?

File: code/modules/economy/worth_items.dm [hunk 2]
```diff
@@ -16,12 +16,14 @@
...
+			if(tech[t] <= 1)
+				continue
+			var/next_tech_val = ((tech[t] - 1)**2) * 5
 			if(next_tech_val > largest_tech_val)
 				largest_tech_val = next_tech_val
-			. += largest_tech_val
+		. += largest_tech_val
 
-	if(force)
+	if(item_flags & ITEM_FLAG_IS_WEAPON && force)
```
Comment (comma) [inline code/modules/economy/worth_items.dm]: (item_flags & ITEM_FLAG_IS_WEAPON) && force

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 3]
```diff
@@ -438,12 +438,13 @@
 
 /obj/item/storage/box/matches/attackby(obj/item/flame/match/W, mob/user)
 	if(istype(W) && !W.lit && !W.burnt)
-		W.lit = 1
-		W.damtype = "burn"
+		W.lit = TRUE
```
Comment (MistakeNot4892) [inline code/game/objects/items/weapons/storage/boxes.dm]: Maybe the lighting/unlighting should be a match proc.

File: code/game/objects/objs.dm [hunk 4]
```diff
@@ -35,7 +35,7 @@
 	return ..()
 
 /obj/proc/get_matter_amount_modifier()
-	. = CEILING(w_class * BASE_OBJECT_MATTER_MULTPLIER)
+	. = w_class * BASE_OBJECT_MATTER_MULTPLIER
```
Comment (MistakeNot4892) [inline code/game/objects/objs.dm:38]: Minor concern, but we should probably do rounding in the places where this is used as a mult now.

Comment (out-of-phaze) [reply code/game/objects/objs.dm:38]: I think we do

Comment (out-of-phaze) [reply code/game/objects/objs.dm:38]: yeah, every usage uses round()

File: code/game/objects/items/weapons/storage/boxes.dm (hunk 3 above)
Comment (out-of-phaze) [reply code/game/objects/items/weapons/storage/boxes.dm]: Unlighting is, it's `extinguish()`

**Review by MistakeNot4892 — APPROVED** [review]

**Pull request merged**